src/Maestro/tests/SubscriptionActorService.Tests/PullRequestActorTests.cs

```diff
@@ -85,7 +85,7 @@ private void ThenGetRequiredUpdatesShouldHaveBeenCalled(Build withBuild)
             DarcRemotes[TargetRepo]
                 .Verify(r => r.GetRequiredNonCoherencyUpdatesAsync(SourceRepo, NewCommit, Capture.In(assets), Capture.In(dependencies)));
             DarcRemotes[TargetRepo]
-                .Verify(r => r.GetDependenciesAsync(TargetRepo, TargetBranch, null));
+                .Verify(r => r.GetDependenciesAsync(TargetRepo, TargetBranch, null, false));
```

Comment:
Nit: could you name these `null` and `false` literals so it'd be easy to read?

Reply:
Can't. Can't use optional params, or named parameters in Expression trees.